e2e/scenario_test.go

```diff
@@ -1795,3 +1794,16 @@ func Test_Ubuntu2404ARM(t *testing.T) {
 		},
 	})
 }
+
+func Test_Ubuntu2204_DiskPressureCondition(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Description: "Validates that DiskPressure condition is false on a newly created Ubuntu 2204 node",
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2204Gen2Containerd,
+			Validator: func(ctx context.Context, s *Scenario) {
+				ValidateNodeConditionStatus(ctx, s, v1.NodeDiskPressure, v1.ConditionFalse)
```

Comment:
sorry for draft review, but why not just add this to common validators instead of creating a dedicated scenario?